content/tracing/setup/go.md

```diff
@@ -5,56 +5,77 @@ aliases:
 - /tracing/go/
 - /tracing/languages/go
 further_reading:
-- link: "https://github.com/DataDog/dd-trace-go"
+- link: "https://github.com/DataDog/dd-trace-go/tree/v1"
   tag: "Github"
   text: Source code
-- link: "https://godoc.org/github.com/DataDog/dd-trace-go/tracer"
+- link: "https://godoc.org/gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
   tag: "GoDoc"
   text: Package page
 - link: "tracing/visualization/"
   tag: "Documentation"
   text: "Explore your services, resources and traces"
 ---
 
-## Installation
+## Getting started
 
-To begin tracing applications written in Go, first [install and configure the Datadog Agent][1] (see additional documentation for [tracing Docker applications](/tracing/setup/docker/)).
+For configuration instructions and details about using the API, check out our [API documentation][api docs] for manual instrumentation, and our [integrations section][contrib docs] for Go libraries and frameworks supporting automatic instrumentation.
 
-Next, install the Go Tracer from the Github repository:
+For a description of the terminology used in APM, take a look at the [Getting started with APM section][getting started]. For details about contributing, check the official repository [README.md file][repo readme]. 
+
+Consult our [migration document][migrating] if you need to migrate from an older version of the tracer (e.g. v<0.6.x) to newest version.
+
+### Requirements
+
+To begin tracing your Go applications, your environment must first meet the following requirements:
+
+* Runing the Datadog Agent >= 5.21.1. See ["Install and configure the Datadog Agent"][1] (additional documentation for [tracing Docker applications](/tracing/setup/docker/)).
+* Using Go 1.9+
+
+### Installation
+
+Next, install the Go tracer from its canonical import path:
 
 ```go
-go get "github.com/DataDog/dd-trace-go/tracer"
+go get gopkg.in/DataDog/dd-trace-go.v1/ddtrace
 ```
 
-Finally, import the tracer and instrument your code!
+You are now ready to import the tracer and start instrumenting your code!
+
+## Manual instrumentation
```

Comment:
nit: sometimes these titles are capitalized and sometimes they're in sentence case.  @l0k0ms is one preferred?  Let's at least be consistent within this doc.

Reply:
I capitalized all of them, like on the [Java page](https://docs.datadoghq.com/tracing/setup/java/#sampling-distributed-tracing) to be consistent.